### ISSUE TITLE
[Enhancement]: Youtube Shopping Integration

### ISSUE THREAD

**Issue by Hpareek07** [woothemes/woocommerce#53723]
### Describe the solution you'd like

Hi Team, 
can you please add youtube shopping integration to woocommerce as currently people are forced to move to shopify for the same. Also, I already saw the open feature requests for the same but there is neither any comment by developers nor any date specified on when it will be done. Can you please clarify on the same.

**Comment by MatthiasReinholz** [comment]
Hi @Hpareek07 , Thanks for sharing your feature request.
Can you describe what specific functionality you're looking for? Are you asking about features such as tagging products on YouTube videos?

**Comment by Hpareek07** [comment]
Yes @MatthiasReinholz , that's the exact feature I am talking about. Currently tagging products in youtube videos is not possible with Woocommerce/Wordpress. If possible can this please be done soon as all other major platforms support it.

**Comment by Hpareek07** [comment]
Hi Team, Any update on this or the timelines ?

**Comment by nerrad** [comment]
Hi @Hpareek07 - thanks for the suggestion! This is something we will be evaluating in the future as a part of an extension, but not something likely to be provides as a core plugin feature.

**Comment by arthurbackouche** [comment]
Hi @nerrad, @MatthiasReinholz 

Everyone is asking for it: https://woocommerce.com/feature-request/add-shop-to-youtube/

Your loyal customers are leaving Woocommerce to join Shopify... Why aren't you a bit more alerted? 

The feature is needed and urgent. Woocommerce is not a tier 1 e-commerce platform anymore if you are not integrating this feature ASAP. This is foundational.

Please listen to your customers and develop the product that people wants.

**Comment by MatthiasReinholz** [comment]
Hi @arthurbackouche,

As this feature is outside the scope of what WooCommerce Core handles, it will be part of the [Google for WooCommerce extension](https://github.com/woocommerce/google-listings-and-ads).

You will find all related updates about such an integration on that repository.
